src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java

```diff
@@ -17,28 +17,48 @@
 package io.cdap.e2e.pages.actions;
 
 import io.cdap.e2e.pages.locators.CdfStudioLocators;
-import io.cdap.e2e.utils.ConstantsUtil;
+import io.cdap.e2e.utils.ElementHelper;
 import io.cdap.e2e.utils.SeleniumDriver;
 import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.e2e.utils.WaitHelper;
 import org.openqa.selenium.By;
-import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.WebElement;
-
-import static io.cdap.e2e.utils.ConstantsUtil.JS_CLICK;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- * Represents CdfStudioActions
+ * Represents Cdf Studio page Actions
  */
 public class CdfStudioActions {
+  private static final Logger logger = LoggerFactory.getLogger(CdfStudioActions.class);
   public static CdfStudioLocators cdfStudioLocators;
 
   static {
     cdfStudioLocators = SeleniumHelper.getPropertiesLocators(CdfStudioLocators.class);
   }
 
+  public static void selectDataPipelineType(String option) {
+    ElementHelper.clickOnElement(CdfStudioLocators.dataPipelineTypeDropdown);
+    ElementHelper.clickOnElementUsingJsExecutor(CdfStudioLocators.locateDataPipelineTypeDropdownOption(option));
```

Comment:
Please open a JIRA for it as discussed offline and add it to the code as a comment. 
Context - using JsExecutor since alert is not getting closed. Also add the screenshot in the JIRA.

Reply:
Added.